mmdnn/conversion/tensorflow/tensorflow_frozenparser.py

```diff
@@ -307,6 +307,16 @@ def _convert_layers_instancenorm(self, source_node):
         # assert False
 
 
+    def _get_node_weight(self, input_node_weight):
+        if input_node_weight.type == 'Const':
+            tensor_content = input_node_weight.get_attr('value')
+        else:
+            input_node_weight_read = self.src_graph.get_parent(input_node_weight.name, [0])
+            tensor_content = input_node_weight_read.get_attr('value')
+        W = tensor_util.MakeNdarray(tensor_content)
+        return W   
+
+
```

Comment:
_get_node_weight is similar to check_const

Reply:
right, reusing check_const() can simplify the codes